plugins/inventory/aws_ec2.py

```diff
@@ -188,6 +188,31 @@
 exclude_filters:
 - tag:Name:
   - 'my_first_tag'
+
+# Example using groups to assign the running hosts to a group based on vpc_id
+plugin: aws_ec2
+boto_profile: aws_profile
+# or you could use Jinja as:
+# boto_profile: "{{ lookup('env', 'AWS_PROFILE') | default('aws_profile', true) }}"
+# Populate inventory with instances in these regions
+regions:
+  - us-east-2
+filters:
+  # All instances with their state as `running`
+  instance-state-name: running
+keyed_groups:
+  - prefix: tag
+    key: tags
+hostnames:
+  - tag:Name
```

Comment:
The `hostnames` block is already covered in some other examples and this is not what you want to demonstrate. You can just remove the key and stick with the default value.